code/mobile/android/PhoneVR/app/src/main/java/viritualisres/phonevr/MainActivity.kt

```diff
@@ -14,8 +14,10 @@ import android.view.WindowManager
 import android.widget.TextView
 import androidx.appcompat.app.AppCompatActivity
 import androidx.appcompat.widget.Toolbar
+import viritualisres.phonevr.Wrap.deinitializeNativeCrashHandler
+import viritualisres.phonevr.Wrap.initializeNativeCrashHandler
 
-import viritualisres.phonevr.BuildConfig;
+//import viritualisres.phonevr.BuildConfig;
```

Comment:
Why commenting BuildConfig, This is required to show VERSION_CODE and VERSION_STR_NAME in home page.